app/presenters/outcome_presenter.rb

```diff
@@ -74,4 +73,19 @@ def render_data_partial(partial, variable_name)
     partial_path = ::SmartAnswer::FlowRegistry.instance.load_path.join("data_partials", "_#{partial}")
     ApplicationController.new.render_to_string(file: partial_path.to_s, layout: false, locals: {variable_name.to_sym => data})
   end
+
+  def render_erb_template
+    unless @rendered_erb_template
+      @view.render(template: erb_template_name, locals: @state.to_hash)
+      @rendered_erb_template = true
+    end
+  end
+
+  def has_content_for_body?
```

Comment:
You're right! I think I probably went with `has_nnn?` as that seems to be the convention in [node_presenter.rb](https://github.com/alphagov/smart-answers/blob/master/app/presenters/node_presenter.rb#L79).

I'm going to leave these as they are for now (as I want to get this merged) but accept that they could be more rubyish.